de-lib/src/main/java/org/iplantc/de/diskResource/client/ToolbarView.java

```diff
@@ -18,7 +18,6 @@
 import org.iplantc.de.diskResource.client.events.selection.RefreshFolderSelected.HasRefreshFolderSelectedHandlers;
 import org.iplantc.de.diskResource.client.events.selection.RenameDiskResourceSelected.HasRenameDiskResourceSelectedHandlers;
 import org.iplantc.de.diskResource.client.events.selection.RestoreDiskResourcesSelected.HasRestoreDiskResourceSelectedHandlers;
-import org.iplantc.de.diskResource.client.events.selection.SaveMetadataSelected.HasSaveMetadataSelectedEventHandlers;
```

Comment:
Since you already had the Handlers set up for this, was there a reason you preferred to switch to the EventBus now?